app/lib/dttp/params/placement_assignment.rb

```diff
@@ -6,6 +6,9 @@ class PlacementAssignment
       include Mappable
 
       ACADEMIC_YEAR_2020_2021 = "76bcaeca-2bd1-e711-80df-005056ac45bb"
+      ACADEMIC_YEAR_2021_2022 = "21196925-17b9-e911-a863-000d3ab0dc71"
+      ACADEMIC_YEAR_2022_2023 = "ed0db9f4-eff5-eb11-94ef-000d3ab1e900"
```

Comment:
In dttp dev this is `"78bcaeca-2bd1-e711-80df-005056ac45bb"`, may cause an issue in qa

Reply:
I got these from prod but have just realised there are duplicates in there. Let me check 'em.

Reply:
I've checked again in the DTTP back office and these are defo correct. There are two 22/23 in the API response in postman though including the one you've said. I think that must be inactive on prod.